pkg/kubectl/resource/builder.go

```diff
@@ -212,6 +212,9 @@ func (b *Builder) Path(recursive bool, paths ...string) *Builder {
 
 		b.paths = append(b.paths, visitors...)
 	}
+	if len(b.paths) == 0 && len(b.errs) == 0 {
+		b.errs = append(b.errs, fmt.Errorf("error reading %v: please make sure the file extension is %v", paths, FileExtensions))
```

Comment:
I would like to give user a more suggestive prompt like `file extension is not [.json .yaml .yml] will be ingnore in directory, please make sure...`